pandas/core/indexes/base.py

```diff
@@ -886,7 +886,7 @@ def astype(self, dtype, copy=True):
 
         Parameters
         ----------
-        indices : list
+        indices : array_like
```

Comment:
```suggestion
        indices : array-like
```

https://pandas.pydata.org/pandas-docs/dev/development/contributing_docstring.html#parameter-types